docs/design-documents/hal/0002-pinmap-extension.md

```diff
@@ -1,6 +1,6 @@
-# HAL  PinMap
+# HAL PinMap design document
 
-<!-- toc -->
+# Table of contents
```

Comment:
You have changed paragraph names, but haven't updated the links below (they don't work).